cutlass/reduction/threadblock_swizzle.h

```diff
@@ -35,7 +35,7 @@ struct DefaultBlockSwizzle {
   CUTLASS_HOST_DEVICE DefaultBlockSwizzle() {}
 
   /// Swizzle the block index.
-  CUTLASS_DEVICE dim3 swizzle() { return blockIdx; }
+  CUTLASS_DEVICE dim3 swizzle() { return {blockIdx.x, blockIdx.y, blockIdx.z}; }
```

Comment:
I guess this change uses a c++11 feature. As state before we would like to avoid that if we can.

Reply:
blockIdx is special. Clang only allows accessing it as individual fields. 
I can rewrite it as a bit more verbose, but equivalent
```
    dim3 r;
    r.x = blockIdx.x;
    r.y = blockIdx.y;
    r.z = blockIdx.z;
    return r;
```
https://godbolt.org/z/zap7ln

Reply:
or could you do `return dim3(blockIdx.x, blockIdx.y, blockIdx.z);`?

Reply:
Indeed. It's been a long day. Will do.

Reply:
Fixed.